app/sdil/controllers/VariationsController.scala

```diff
@@ -54,15 +55,15 @@ class VariationsController(
   registeredAction: RegisteredAction,
   cache: ShortLivedHttpCaching,
   errorHandler: FrontendErrorHandler,
-  mcc: MessagesControllerComponents
-)(
-  implicit
-  val config: AppConfig,
-  val ec: ExecutionContext)
-    extends FrontendController(mcc) with SdilWMController with FormHelpers with ReturnJourney {
+  mcc: MessagesControllerComponents,
+  uniformHelpers: Uniform,
+  config: AppConfig
+)(override implicit val ec: ExecutionContext)
```

Comment:
You could move the appConfig here, then you don't have to assign the val further down